deploy-manage/uninstall/uninstall-elastic-cloud-enterprise.md

```diff
@@ -10,7 +11,7 @@ You might need to remove Elastic Cloud Enterprise for one of the following reaso
 * If the installation process does not complete successfully and you cannot troubleshoot the issue.
 * If you are de-provisioning a host and want to remove the installed Elastic Cloud Enterprise software.
 
-You remove Elastic Cloud Enterprise by removing all containers on the host:
+You can remove Elastic Cloud Enterprise by removing all containers on the host:
 
 * If using Docker
```

Comment:
the note on line 35 of this page makes the scope of this page confusing ... do these instructions uninstall the orchestrator, or just orchestrated deployments?

Reply:
Good point @shainaraskas.

The scope of that doc is to Uninstall an ECE host from an ECE platform.
It's going to destroy all associated containers, no matter if they are ECE platform related or deployments related.

We can do a quick and beneficial improvement to this page:
- Add a disclaimer warning with a link to "vacate allocators", like:

```
::::{warning}
if the ECE host to uninstall has the allocator role and its running instances from orchestrated deployments, the containers will be deleted and the instances will appear unhealthy in the deployments page. In this case it's recommended to [vacate the ECE host](/deploy-manage/maintenance/ece/move-nodes-instances-from-allocators) before uninstalling ECE.
::::
```